src/MDFT.jl

```diff
@@ -0,0 +1,108 @@
+"""
+    MDFT{T<:Real} <: Simulator
+
+A model type for Multialternative Decision Field Theory. 
+    
+# Parameters 
+
+- `ν = [2.5,2.0]`: drift rates 
+- `α = 1.5`: evidence threshold 
+- `β = .20`: lateral inhabition 
+- `λ = .10`: leak rate
+- `τ = .30`: non-decision time 
+- `σ = 1.0`: diffusion noise 
+- `Δt = .001`: time step 
+
+# Constructors 
+
+    MDFT(ν, α, β, λ, τ, σ, Δt)
+
+    MDFT(;ν = [2.5,2.0], 
+        α = 1.5, 
+        β = .20, 
+        λ = .10, 
+        τ = .30, 
+        σ = 1.0, 
+        Δt = .001)
+        
+# Example 
+
+```julia 
+using SequentialSamplingModels 
+ν = [2.5,2.0]
+α = 1.5
+β = 0.20
+λ = 0.10 
+σ = 1.0
+τ = 0.30
+Δt = .001
+
+dist = MDFT(; ν, α, β, λ, τ, σ, Δt)
+choices,rts = rand(dist, 500)
+```
+# References
+
+Usher, M., & McClelland, J. L. (2001). The time course of perceptual choice: The leaky, competing accumulator model. Psychological Review, 108 3, 550–592. https://doi.org/10.1037/0033-295X.108.3.550
```

Comment:
is that the right ref for MDFT?

Reply:
Good catch. I copied over code from the LCA. The doc strings will be fixed in the next commit now that the code is producing the correct output.